src/pipelines/html.rs

```diff
@@ -109,9 +109,9 @@ impl HtmlPipeline {
 
         // Assemble a new output index.html file.
         let output_html = target_html.html(); // TODO: prettify this output.
-        fs::write(format!("{}/index.html", self.cfg.dist.display()), output_html.as_bytes())
+        fs::write(self.cfg.dist.join("index.html"), output_html.as_bytes())
```

Comment:
☠️ wow ... sorry @MartinKavik that's definitely my fault. Good find! Sorry about the trouble.